yorkie/src/main/kotlin/dev/yorkie/document/Document.kt

```diff
@@ -182,6 +201,13 @@ public class Document(public val key: Key) {
         return root.garbageCollect(ticket)
     }
 
+    /**
+     * Updates the status of this [Document].
+     */
+    public fun setStatus(status: DocumentStatus) {
```

Comment:
I don't think we need a separate set method for status.
let's just make status property's setter public or internal.
(I think it should be internal)